Update Using QCity documentation - building levels (#66)
north-road/QCity: docs/src/user_guide/tools.md

![Project_Settings_Rename_Project_Area](https://github.com/user-attachments/assets/4ea00ba2-a6e5-499e-9432-a9999ac81cb0)

A new window will open which will allow you to rename the project area.

![Project_Area_Rename_Window](https://github.com/user-attachments/assets/111caf57-77cd-4d12-8048-fbc733dde0bb)

> > Should a name already be used, QCity will warn you that the name already exists and will not allow you to use that name.

Reviewer: ```suggestion
> Should a name already be used, QCity will warn you that the name already exists and will not allow you to reuse it.
```

![Snapping_Indicator](https://github.com/user-attachments/assets/0120e5af-4b45-4a2a-b97c-1e9487a403f5)

> For more information on using snapping within QGIS, please see the [QGIS documentation](https://docs.qgis.org/latest/en/docs/user_manual/working_with_vector/editing_geometry_attributes.html#snapping-and-digitizing-options).

### Deleting a Development Site

> A development site can be deleted by left clicking on the development site you want to delete in the development sites panel and left clicking on the red minus button on the right hand side of the development sites panel.

Reviewer: ```suggestion
A development site can be deleted by selecting the development site you want to delete in the development sites panel and left clicking on the red minus button on the right hand side of the development sites panel.
```

![Development_Sites_Delete](https://github.com/user-attachments/assets/7ce818b2-044e-4bec-9104-ed6bfb2aac63)

**Caution: Deleting a development site will also delete any associated building levels.**

### Renaming a Development Site

> A development site can be renamed by left clicking on the development site you want to rename in the development sites panel and left clicking on the pencil and paper button on the right hand side of the development sites panel.

Reviewer: ```suggestion
A development site can be renamed by selecting the development site you want to rename in the development sites panel and left clicking on the pencil and paper button on the right hand side of the development sites panel.
```

![Development_Site_Rename](https://github.com/user-attachments/assets/40c66d57-053c-4df0-8b29-2821c7e5b572)

A new window will open which will allow you to rename the development site.

![Development_Site_Rename_Window](https://github.com/user-attachments/assets/5139b1b6-111b-4e1f-b125-628ceaed7e66)

> > Should a name already be used, QCity will warn you that the name already exists and will not allow you to use that name.

Reviewer: ```suggestion
> Should a name already be used, QCity will warn you that the name already exists and will not allow you to reuse it.
```

![Snapping_Advanced_Configuration](https://github.com/user-attachments/assets/b10886f2-1eab-4a1c-a6d3-26c94ad44430)

Once snapping has been enabled, when created a new building level, you will see a purple indicator on the map canvas to snap to the desired features.

![Snapping_Indicator](https://github.com/user-attachments/assets/0120e5af-4b45-4a2a-b97c-1e9487a403f5)

> For more information on using snapping within QGIS, please see the [QGIS documentation](https://docs.qgis.org/latest/en/docs/user_manual/working_with_vector/editing_geometry_attributes.html#snapping-and-digitizing-options).

### Creating a new Building Level - Advanced Digitizing

When digitizing building levels you may wish to digitize with specific measurements and angles in order to create polygons with accurate setbacks and lengths. In order to digitize with this level of accuracy, you will need to use the QGIS Advanced Digitizing Tools.

> > **Note: QGIS Advanced Digitizing tools only work with QCity in QGIS 3.44 or newer. In older QGIS releases, the Advanced Digitizing tools will not work when attempting to digitize project areas, development sites or building levels from within QCity.**

Reviewer: We bumped the minimum version to 3.44 to work around this -- I think this line could be dropped

Author: Yes. Happy to drop this bit.

> 
> > **Note: QGIS Advanced Digitizing tools only work with QCity in QGIS 3.44 or newer. In older QGIS releases, the Advanced Digitizing tools will not work when attempting to digitize project areas, development sites or building levels from within QCity.**

Reviewer: ```suggestion
```

4. Clear the construction guides completely

> The construction mode can also be enable and disabled with the keyboard shortcut `c` when using the Advanced Digitizing tools.

Begin creating a new building level polygon by left clicking on the map area. With the Advanced Digitizing tools enabled, you can input the distance and angle you would like to create the first length of the polygon.

![Advanced_Digitizing_Distance_and_Angle](https://github.com/user-attachments/assets/e3a1d9c5-46d7-45d5-a473-7c3f6d3ce7ec)

Once you had created the first point of the polygon, enter the distance and angle and left click again to create the next point. Repeat this process and right click to finish the polygon. Once you complete the building level polygon, a window will appear asking for a name of the building level polygon as outlined in [Creating a Building Level](#creating-a-new-building-level).

> For more information on the QGIS Advanced Digitizing tools, please see the [QGIS Documentation](https://docs.qgis.org/latest/en/docs/user_manual/working_with_vector/editing_geometry_attributes.html#the-advanced-digitizing-panel)

### Deleting a Building Level

> A building level can be deleted by left clicking on the building level you want to delete in the building levels panel and left clicking on the red minus button on the right hand side of the building levels panel.

Reviewer: ```suggestion
A building level can be deleted by selecting the building level you want to delete in the building levels panel and left clicking on the red minus button on the right hand side of the building levels panel.
```

![Building_Levels_Delete](https://github.com/user-attachments/assets/ce491c6e-06ab-4b2f-ab2b-57f62fb7ec5d)

### Renaming a Building Level

> A building level can be renamed by left clicking on the building level you want to rename in the building level panel and left clicking on the pencil and paper button on the right hand side of the building level panel.

Reviewer: ```suggestion
A building level can be renamed by selecting the building level you want to rename in the building level panel and left clicking on the pencil and paper button on the right hand side of the building level panel.
```

![Building_Levels_Rename](https://github.com/user-attachments/assets/9ef4b78b-435b-463a-ab98-c4caa03acc2a)

A new window will open which will allow you to rename the building level.

![Building_Levels_Rename_Window](https://github.com/user-attachments/assets/19041748-6279-4691-a72a-d2edc6a3a549)

> > Should a name already be used, QCity will warn you that the name already exists and will not allow you to use that name.

Reviewer: ```suggestion
> Should a name already be used, QCity will warn you that the name already exists and will not allow you to reuse it.
```
